nss-day-cohort-45/tabloid-cli-nights-court
# Implemented Prelim Soft Delete for Posts, Authors, Tags

### Changes:
1. Updated SQL SELECT queries in Posts, Authors, Tags repositories to check only for non-deleted items
2. Changed DELETE queries to UPDATE queries that change the IsDeleted bit to 1
---
### Testing:
1. Fetch branch, open app (after having run the new SQL database creation script)
2. Select option 4 for Post Management
3. Select option 3 to add a test post
4. List posts (option 1) to view all posts
5. Select option 5 to remove a post
6. View all posts again to see that it was deleted.
7. Repeat the same testing for Authors and for Tags
